File: python/exercises/exercise05.py
import os
os.system("cls||clear")

pares = 0
impares = 0

for i in range(1,6):
    numeros = int(input("Digite um número inteiro: "))
    
    if numeros % 2 == 0:
        pares += 1
    else:
        impares += 1

print(f"Pares: {pares}")
print(f"Impares: {impares}")

File: c/exercises/exercise05.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main(){
	setlocale(LC_ALL, "");
	
	float salario, salariosMinimos;
	
	printf("Digite o seu salário: ");
	scanf("%f",&salario);
	
	salariosMinimos = salario / 1412;
	
	system("cls||clear");
	printf("=== Exibindo resultados ===\n");
	printf("Salário: %3.f \n", salario);
	printf("Salarios mínimos: %f \n", salariosMinimos);
	
	return 0;
}
